index.js

```diff
@@ -246,24 +302,165 @@ app.delete('/artists/:id/', (request, response) => {
     });
 });
 
+// GET Method - get form to show all playlists
+app.get('/playlist/', (request, response) => {
+
+    // Construct query to select all playlist from database using pool.query
+    const getAllPlaylists = 'SELECT * FROM playlist';
+
+    pool.query(getAllPlaylists, (err, result) => {
+        if (err) {
+            console.log("Error: ", err.message);
+        } else {
+            // Store the result as data object
+            const data = {
+                playlists: result.rows
+            };
+
+            response.render('allPlaylist', data);
+        }
+    });
+});
+
+// GET Method - get form to create new playlist
+app.get('/playlist/new', (request, response) => {
+
+    response.render('newPlaylist');
+});
+
+// POST Method - Save new playlist
+app.post('/playlist', (request, response) => {
+
+    // Get the data from request body
+    let newPlaylistName = request.body.playlistName;
+
+    const inputValues = [newPlaylistName];
+
+    // Construct insert query
+    const createPlaylist = 'INSERT INTO playlist (name) VALUES ($1)';
+
+    // Call pool.query to save the request body to database
+    pool.query(createPlaylist, inputValues, (err, result) => {
+        if (err) {
+            console.log("Error: ", err.message);
+        } else {
+            response.send("Playlist created successfully.");
+        }
+    });
+
+});
+
+// GET Method - get form to show details of individual playlist
+app.get('/playlist/:id', (request, response) => {
+
+    let playlistId = request.params.id;
+
+    // Construct query to select the specified playlist from database using pool.query
+    const getSpecificPlaylist = 'SELECT title FROM songs INNER JOIN playlist_song ON (songs.id = playlist_song.song_id) WHERE playlist_id = ' + playlistId;
```

Comment:
try to use $1,$2 instead for values in sql query

Reply:
👌 Updated. Thank u :)